test/Microsoft.AspNetCore.Server.Kestrel.Performance/Microbenchmarks.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using BenchmarkDotNet.Attributes;
+using Microsoft.AspNetCore.Server.Kestrel.Internal.Infrastructure;
+using System;
+using System.Text;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Performance
+{
+    public class KnownStrings
+    {
+        static byte[] _method = Encoding.UTF8.GetBytes("GET ");
+        static byte[] _version = Encoding.UTF8.GetBytes("HTTP/1.1\r\n");
+
+        [Benchmark]
```

Comment:
Since you are actually doing 10k parses per function you want this attribute to be
`[Benchmark(OperationsPerInvoke = 10000)]`
so it knows to divide down the time.